packages/react/kibana_context/root/README.mdx

```diff
@@ -0,0 +1,47 @@
+---
+id: react/context/root
+slug: /react/context/root
+title: React Context - Root
+description: This context provider is used only used by the very base root of Kibana.  Unless you're writing tests, a Storybook, or working in core code, you likely don't need this.
+tags: ['shared-ux', 'react', 'context']
+date: 2023-07-25
+---
+
+## Description
+
+This package contains a root context provider for Kibana rendering.  It handles operations that should only happen _once_ when the browser loads a page.
+
+While it would be safer to isolate this in a `core` package, we need to use it in other contexts-- like Storybook and Jest.
```

Comment:
Just an optional nit, for core we've had a pattern of naming packages like this with an `-internal` suffix in the id & directory, to avoid confusion about what's safe to use. Could consider something similar here, `@kbn/react-kibana-context-root-internal`

Reply:
The thing is, this context is also useful in some edge cases.  For example, the Fatal Error screen empties the entire page and rebuilds it from scratch... so the root context is useful there.

I'm thinking a lot about answering "hey, I have an entire page or tool, what context do I use?" questions... and the copy/paste that happens when code is "internal".

Reply:
Ah, fatal error is a great example which I had overlooked. This makes sense then. As long as folks understand this is most likely not the component they are looking for, this should be sufficient.